migrations/migrations.sql

```diff
@@ -4303,5 +4303,17 @@ do $$
     if not exists (select 1 from information_schema.columns where table_name = 'accountIntegrations' and column_name = 'isInternal') then
       alter table "accountIntegrations" add column "isInternal" BOOLEAN;
     end if;
+
+    -- Add projectId column to jobTestReports table and populate projectId for existing jobTestReports
+    if not exists (select 1 from information_schema.columns where table_name = 'jobTestReports' and column_name = 'projectId') then
+      alter table "jobTestReports" add column "projectId" varchar(24);
+      update "jobTestReports" SET "projectId" = jobs."projectId" from jobs where "jobTestReports"."jobId" = "jobs".id;
```

Comment:
Just noticed -- do we need a null check here so this doesn't happen every time?

Reply:
This happens only when the column is being added, right? It can't happen again after that.

Reply:
yes it is a one time run, only when the projectId field is not there

Reply:
Yep, missed that.